### PR TITLE
New documentation2

### PR DESCRIPTION
**Pull request by arwelHughes**
Added documentation for Utility Functions.

### CODE REVIEW

File: examples/normalReflectivity/standardLayers/newDirectory/datafiles/newFile04-12-2024 11.48.23.274Datafile1
Comment (StephenNneji) [inline examples/normalReflectivity/standardLayers/newDirectory/datafiles/newFile04-12-2024 11.48.23.274Datafile1:1]: Are these files needed? Please can you remove them if not?

File: docs/.idea/workspace.xml
Comment (StephenNneji) [inline docs/.idea/workspace.xml:1]: This folder is in the _.gitignore_, how did you manage to add it to the branch :grin:?

File: docs/source/conversionFuncs.rst [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
 
-
+===============================
 Convert between RAT and RasCAL1
-...............................
+===============================
+
+Two utilities are provided to convert between RasCAL1 and RAT projects:
+
+* **Convert from R1 to RAT** : You can convert any R! project directly to a projectClass ana analyse using RAT
```
Comment (StephenNneji) [inline docs/source/conversionFuncs.rst:9]: typos: 

- R1 instead of R!
- and instead of ana

File: docs/source/plotFuns.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    figure(1); clf
+    plotRefSLD(problem,results)
+
+.. image:: images/misc/simPlot1.png
+    :alt: disp(results)
+
+
+This produces a basic plot of the reflectivity and SLD.
+
+If this plot is not cleared before subsequent plots, then *potRefSLD* will overplot allowing the results of calculations to be compared:
```
Comment (StephenNneji) [inline docs/source/plotFuns.rst:21]: typo 

- plotRefSLD instead of potRefSLD

File: docs/.idea/workspace.xml
Comment (arwelHughes) [reply docs/.idea/workspace.xml:1]: No idea!!! :)

File: examples/normalReflectivity/standardLayers/newDirectory/datafiles/newFile04-12-2024 11.48.23.274Datafile1
Comment (arwelHughes) [reply examples/normalReflectivity/standardLayers/newDirectory/datafiles/newFile04-12-2024 11.48.23.274Datafile1:1]: Probably not - please remove..

**Review by StephenNneji — APPROVED** [review]
